segmentstore/storage/src/main/java/io/pravega/segmentstore/storage/chunklayer/WriteOperation.java

```diff
@@ -265,8 +265,8 @@ private void collectGarbage() {
                     didSegmentLayoutChange = true;
                     chunksAddedCount.incrementAndGet();
 
-                    log.debug("{} write - New chunk added - segment={}, chunk={}, offset={}.",
-                            chunkedSegmentStorage.getLogPrefix(), handle.getSegmentName(), newChunkName, segmentMetadata.getLength());
+                    log.debug("{} write - New chunk added - op={}, segment={}, chunk={}, offset={}.",
+                            chunkedSegmentStorage.getLogPrefix(), System.identityHashCode(this), handle.getSegmentName(), newChunkName, segmentMetadata.getLength());
```

Comment:
how is this helpful to identify it? Also, what's wrong with `this.hashcode()`?

Reply:
I needed some unique id for for each request.
Each operation invocation is a new object and that is good enough to for debug purpose.

System.identityHashCode(this) is more "stable", can't be overridden by implementation and more likely to be unique than 
HashCode.